### PR TITLE
Landing style

### PR DESCRIPTION
**Pull request by TopCodeAndrew**
Full responsive design in landing page and login popup! In working through this, I cleaned up our scss file structure, changed a little bit of wording, and removed the description paragraph in the "request access" section.

### CODE REVIEW

File: src/Components/Landing/Login.js [hunk 1]
```diff
@@ -47,24 +47,28 @@ const Login = props => {
...
+        onClick={(e) => togglePopup()}
+      />
+
+
+      {/* THIS IS THE POPUP THAT APPEARS WHEN "LOGIN" IS CLICKED ON THE LANDING PAGE */}
+      {isOpen ? <Popup content={<>
         <div className="popup-header">
           {/* <img className="logo" src={logo} ></img> */}
-          <h2>Lemon Prop Mgmt</h2>
+          <h2>Lemon Prop Login</h2>
```
Comment (brookecm28) [inline src/Components/Landing/Login.js:60]: We should remove the space between Lemon and Prop here

Comment (TopCodeAndrew) [reply src/Components/Landing/Login.js:60]: great catch!

Comment (TopCodeAndrew) [reply src/Components/Landing/Login.js:60]: Are you good to do that when we approve the pull request?

**Review by brookecm28 — APPROVED** [review]

File: src/Components/Landing/Login.js (hunk 1 above)
Comment (brookecm28) [reply src/Components/Landing/Login.js:60]: I just merged. If you're going to be working on it today, I'd say you go ahead and do it. If it's still there when I get back to it next, I'll make sure it gets taken care of for sure.

Comment (TopCodeAndrew) [reply src/Components/Landing/Login.js:60]: you rock!

**Pull request merged**